src/interfaces/UploadResponse.ts

```diff
@@ -0,0 +1,69 @@
+/**
+ * Type of files to include in result set. Accepts three values:
+ * all - include all types of files in result set
+ * image - only search in image type files
+ * non-image - only search in files which are not image, e.g., JS or CSS or video files.
+ *
+ * @see {@link https://docs.imagekit.io/api-reference/media-api/list-and-search-files}
+ */
+export type FileType = "all" | "image" | "non-image";
+
+/**
+ * Response from uploading a file
+ *
+ * @see {@link https://docs.imagekit.io/api-reference/upload-file-api/server-side-file-upload#response-code-and-structure-json}
+ */
+export interface UploadResponse {
+  /**
+   * Unique fileId. Store this fileld in your database, as this will be used to perform update action on this file.
+   */
+  fileId: string;
+  /**
+   * The name of the uploaded file.
+   */
+  name: string;
+  /**
+   * The URL of the file.
+   */
+  url: string;
+  /**
+   * In case of an image, a small thumbnail URL.
+   */
+  thumbnailUrl: string;
+  /**
+   * Height of the uploaded image file. Only applicable when file type is image.
+   */
+  height: number;
```

Comment:
Are we sure the API returns Number JS values? if they're string, i.e. "123", then will it be coerced to TS's number?

Reply:
Yes, they are returned as number as per the documentation
https://docs.imagekit.io/api-reference/upload-file-api/server-side-file-upload